fuel-tx/src/transaction/types/create.rs

```diff
@@ -237,25 +242,29 @@ impl FormatValidityChecks for Create {
     fn check_without_signatures(
         &self,
         block_height: BlockHeight,
-        parameters: &ConsensusParameters,
+        tx_params: &TxParameters,
```

Comment:
If we need a combination of the parameters, it is better to pass only `ConsensusParameters`. It is not a problem if some fields are not used. The API should be simple for the user and transparent regarding requirements(like you don't need to pass `ConsensusParameters` if you only need a `ChainId`).

Reply:
For sure. I've combined them back together in some other places. Clippy just didn't catch this because it's at the limit (7). I can do the same here. Good catch.